testing/functional_testing/fire/ros/ros_test.py

```diff
@@ -0,0 +1,226 @@
+                "reaction_vel": data.reaction_velocity.values.flatten(),
+            }
+        )
+        
+        data_frame['SAV_ft'] = data_frame.SAV*30.48
+
+        max_beta = data_frame["beta_ratio"].max()
+        max_reaction_vel = 18
+        
+        blank_plot(max_beta, 0.0, max_reaction_vel, 0.0, draw_horizontal_lines=True)
+
+        SAV_vals = np.unique(data_frame.SAV_ft.values)
+        colors = ['#793922', '#6B8939', '#99291F','#CC9728', '#2C778A']
+        colors.reverse()
+        
+        for i, sav in enumerate(SAV_vals):
+            dat = data_frame[data_frame.SAV_ft == sav]
+            plt.plot(
+                dat.beta_ratio.values,
+                dat['reaction_vel'].values,
+                lw=2,
+                color=colors[i],
+                label=sav,
+            )
+
+        plt.xlabel("Relative packing ratio", fontsize=11)
+        plt.ylabel("Reaction velocity (min$^{-1}$)", fontsize=11)
+        plt.legend(loc="upper right", title="Surface-area-to-volume ratio (ft$^{-1}$)")
+
+        if save_fig:
+            fig_name = os.path.join(plot_dir, "reaction_vel_plot.png")
+            plt.savefig(fig_name)
+    
+    @staticmethod    
+    def plot_qig(data: xr.Dataset, save_fig: bool, plot_dir: str = None):
+        """Plot heat of preignition
+
+        Args:
+            data (xarray DataSet): the data set 
+            save_fig (bool): whether or not to write out plot
+            plot_dir (str): if saving figure, where to write to
+        """
+        data_frame = pd.DataFrame(
+            {
+                "fuel_moisture": data.fuel_moisture,
+                "q_ig": data.q_ig.values.flatten(),
+            }
+        )
+        
+        data_frame['fuel_moisture_perc'] = data_frame.fuel_moisture*100.0
+        data_frame['q_ig_btu'] = data_frame.q_ig*0.947817/2.20462
+
+        max_moist = 200.0
+        max_qig = 2500.0
+        
+        blank_plot(max_moist, 0.0, max_qig, 0.0, draw_horizontal_lines=True)
+
+        plt.plot(
+            data_frame.fuel_moisture_perc.values,
+            data_frame['q_ig_btu'].values,
+            lw=2,
+            color='k',
+        )
+
+        plt.xlabel("Fuel moisture (%)", fontsize=11)
+        plt.ylabel("Heat of Preignition (Btu lb$^{-1}$)", fontsize=11)
+
+        if save_fig:
+            fig_name = os.path.join(plot_dir, "qig_plot.png")
+            plt.savefig(fig_name)
+            
+    @staticmethod    
+    def plot_eps(data: xr.Dataset, save_fig: bool, plot_dir: str = None):
+        """Plot effective heating number
+
+        Args:
+            data (xarray DataSet): the data set 
+            save_fig (bool): whether or not to write out plot
+            plot_dir (str): if saving figure, where to write to
+        """
+        data_frame = pd.DataFrame(
+            {
+                "SAV": data.SAV,
+                "eps": data.eps.values.flatten(),
+            }
+        )
+        
+        data_frame['SAV_ft'] = data_frame.SAV*30.48
```

Comment:
What's this magic number?

Reply:
whoops cms to ft... will make this a constant and define.